src/Realizer.js

```diff
@@ -7,24 +7,22 @@ class Realizer
 		this.board = game.board;
 		this.isFullyUpdated = false;
 		this.moveQueue = [];
-		this.moveQueueMutex = false;
 		
 		this.cellsPerRow = Math.round(Math.sqrt(this.board.contents.length));
 		
 		this.displayBoard = this.CreateDisplayBoard();
 	}
 	
 	/**
-	 * Method to be called by the front-end. Can be called multiple
-	 * times without incurring computational cost if no update is
-	 * available. Outputs the current state of the board.
+	 * Method to be called by the front-end. Incurs computational
+	 * cost each time called. Outputs the current state of the board.
 	 */
 	Realize()
 	{
-		if (this.isFullyUpdated)
-		{
-			return;
-		}
+		/* Update the board */
+		this.displayBoard = this.CreateDisplayBoard();
```

Comment:
With these changes, I don't think it's necessary to keep `this.displayBoard`. I don't think it is used anywhere else.